CONTRIBUTING.md

```diff
@@ -0,0 +1,48 @@
+# Contribution to NovelCOVID  API
+
+Thanks for taking the time to contribute!
+Every commit supports the open source ecosystem in case of [COVID-19](https://en.wikipedia.org/wiki/2019%E2%80%9320_coronavirus_pandemic).
+
+We love your input! We want to make contributing to this project as easy and transparent as possible, whether it's:
+
+- Reporting a bug
+
+- Discussing the current state of the code
+
+- Submitting a fix
+
+- Proposing new features
+
+
+## Testing
+
+We have a handful of unit tests to cover most of functions.
+Please write new test cases for new code you create.
+
+## Submitting changes
+
+* If you're unable to find an open issue, [open a new one](https://github.com/NovelCOVID/API/issues/new). Be sure to include a **title and clear description**, as much relevant information as possible.
+* Open  a new [GitHub Pull Request to API](https://github.com/NovelCOVID/API/pulls) with a clear list of what you've done (read more about [pull requests](http://help.github.com/pull-requests/)). Include the relevant issue number if applicable.
+* We will love you forever if you include unit tests. We can always use more test coverage
+
+Unsure where to begin contributing to API ? You can start by looking through these issues labels:
+
+* [Enhancement issues](https://github.com/ExpDev07/coronavirus-tracker-api/labels/enhancement) - issues for new feature or request
```

Comment:
seems like you forgot to change the link from the template :P

Reply:
Ooh sorry.
Let's me make changes and submitted it again.